### PR TITLE
create Irish language option

### PR DESCRIPTION
**Pull request by LornitaR**

### CODE REVIEW

**Comment by LornitaR** [comment]
Adding Irish option

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: language.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ﻿If you speak English, [go here](english/marshmallow.md).
 
+Má labhraíonn tú Gaeilge, [téigh anseo](irish/start).
```
Comment (casey-collab) [inline language.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: language.md (hunk 1 above)
Comment (cbuckey-uda) [inline language.md]: Did you maybe forget to add and commit the new file?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**